### DISCUSSION TITLE
Website code examples are not readable

### DISCUSSION THREAD

**Discussion by grundmanise** [pmndrs/valtio#1019] [Bug report] [answered]
### Bug Description

In light mode, the code examples on the website are unreadable (Chrome, Firefox):
<img width="467" alt="image" src="https://github.com/user-attachments/assets/0d8fa1ee-0337-4111-8973-6433440687e8" />

### Reproduction Link

https://valtio.dev/

**Comment by dai-shi** [comment]
Thanks for reporting.
@overthemike Can you look into it?

**Reply by overthemike** [reply]
Yeah I'll take a look

**Reply by overthemike** [reply, marked as answer]
#1017
